nixos/tests/n8n.nix

```diff
@@ -0,0 +1,25 @@
+import ./make-test-python.nix ({ lib, ... }:
+
+with lib;
+
+let
+  port = 5678;
+in
+{
+  name = "n8n";
+  meta.maintainers = with maintainers; [ freezeboy ];
+
+  nodes.machine =
+    { pkgs, ... }:
+    {
+      services.n8n = {
+        enable = true;
+      };
+    };
+
+  testScript = ''
```

Comment:
Is there anything else we can easily demonstrate?

Reply:
currently lacking inpiration

Reply:
I'm not familiar with this software or what it does, hence have nothing of value to say here... so we'll leave it at that 😄